projects/solid/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@maskito/solid",
+    "version": "0.0.1",
+    "description": "",
+    "license": "MIT",
+    "author": "",
+    "type": "module",
+    "module": "dist/index.js",
+    "scripts": {
+        "start": "jiti ./src/index.ts",
+        "build": "tsup"
+    },
+    "devDependencies": {
+        "jiti": "^1.21.0",
+        "solid-js": "^1.8.22",
```

Comment:
Should we also add `solid-js` to [peerDependencies](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependencies) ?